test/integration/users.test.js

```diff
@@ -121,24 +121,22 @@ describe("Users", function () {
         });
     });
 
-    it("Should update the user roles", function (done) {
+    it("Should not update the user roles", function (done) {
       chai
         .request(app)
         .patch("/users/self")
         .set("cookie", `${cookieName}=${jwt}`)
         .send({
           roles: {
-            archived: false,
-            in_discord: false,
-            developer: true,
+            maven: true,
           },
         })
         .end((err, res) => {
           if (err) {
             return done(err);
           }
 
-          expect(res).to.have.status(204);
+          expect(res).to.have.status(403);
```

Comment:
why are we changing the status code from 204 to 403 ?

Reply:
because we are not allowing user to update their role